app/src/main/java/org/traccar/client/MainActivity.java

```diff
@@ -307,4 +291,47 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    private boolean setServerURL(String userUrl) {
+        EditTextPreference preference = (EditTextPreference) findPreference(KEY_URL);
+
+        if (userUrl == null || userUrl.trim().length() == 0) {
+            preference.setText(getString(R.string.settings_url_default_value));
+            preference.setSummary(R.string.settings_url_summary);
+            findPreference(KEY_STATUS).setEnabled(true);
```

Comment:
If an invalid URL had been entered before, Status preference was greyed out. Here it is re-enabled since default URL is obviously valid.